src/overlays/actors/ovl_En_Fishing/z_en_fishing.c

```diff
@@ -4179,19 +4179,20 @@ void EnFishing_UpdateFish(Actor* thisx, PlayState* play2) {
 s32 EnFishing_FishOverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx) {
     EnFishing* this = THIS;
 
-    if (limbIndex == FISHING_FISH_LIMB_0D) {
+    if (limbIndex == FISHING_BASS_LIMB_JAW) {
```

Comment:
With ` FISH` -> `BASS`, could you change `EnFishing_FishOverrideLimbDraw` -> `EnFishing_BassOverrideLimbDraw` and `EnFishing_FishPostLimbDraw` -> `EnFishing_BassPostLimbDraw`